db/migrate/20250827220507_create_browser_color_preference_table.rb

```diff
@@ -0,0 +1,16 @@
+class CreateBrowserColorPreferenceTable < ActiveRecord::Migration[7.2]
+  def change
+    create_table "land.browser_color_preferences", id: :uuid, default: 'gen_random_uuid()', primary_key: :browser_color_preference_id do |t|
+      t.boolean :dark_mode, index: true
+      t.boolean :light_mode, index: true
+      t.boolean :no_preference, index: true
+
+      t.timestamps
+    end
+
+    add_index "land.browser_color_preferences", [:dark_mode, :light_mode, :no_preference], name: 'index_browser_color_preferences_on_color_scheme_preferences'
+
+    add_column "land.user_agents", :browser_color_preference_id, :uuid
+    add_index "land.user_agents", :browser_color_preference_id, name: 'index_user_agents_on_browser_color_preference_id'
```

Comment:
Also, lets use this pattern for index names, as this is what I will be using when updating the databases for UUIDs:
`"#{table_name}_#{column_names.join("_")}_idx`

https://github.com/cpcmarketing/billdoctor-web/pull/1923/files#diff-fbfeaf9f3f4f764b8619d9489a5b120a1ba34ef88617be02433361982593f893R40

Reply:
Here are some options that still follow the pattern, but abbreviate preferences to meet the character limit.
```
browser_color_prefs_dark_light_no_pref_idx (42 chars)
browser_clr_prefs_dark_mode_light_mode_no_preference_idx (57 chars)
brwsr_color_prefs_dark_mode_light_mode_no_preference_idx (57 chars)
browser_color_preferences_dark_mode_light_mode_no_pref_idx(58 chars)
```